lib/string.fz

```diff
@@ -593,3 +593,7 @@ strings is
   zChar     => "z".utf8.asStream.next
   capAChar  => "A".utf8.asStream.next
   capZChar  => "Z".utf8.asStream.next
+
+
+  # a line break
+  line_separator => fromBytes [u8 10]
```

Comment:
Can you write a feature `ascii` (or similarly named) that declares all the ASCII control sequences as `u8` constants and maybe also as `string` such that one could use `ascii.lf`or `ascii.lf_str` (I you find better names, feel free to suggest other names)?